app/assets/javascripts/constants/article_finder.js

```diff
@@ -0,0 +1,14 @@
+export const CLEAR_FINDER_STATE = 'CLEAR_FINDER_STATE';
+export const RECEIVE_CATEGORY_RESULTS = 'RECEIVE_CATEGORY_RESULTS';
+export const RECEIVE_ARTICLE_PAGEVIEWS = 'RECEIVE_ARTICLE_PAGEVIEWS';
+export const RECEIVE_ARTICLE_PAGEASSESSMENT = 'RECEIVE_ARTICLE_PAGEASSESSMENT';
+export const RECEIVE_ARTICLE_REVISION = 'RECEIVE_ARTICLE_REVISION';
+export const RECEIVE_ARTICLE_REVISIONSCORE = 'RECEIVE_ARTICLE_REVISIONSCORE';
+
+export const fetchStates = {
+ TITLE_RECEIVED: 1,
+ PAGEASSESSMENT_RECEIVED: 2,
+ REVISION_RECEIVED: 3,
+ REVISIONSCORE_RECEIVED: 4,
+ PAGEVIEWS_RECEICED: 5
+};
```

Comment:
@ragesoss I added this object to map fetchState to an integer for comparing. Redux state will save the keys of this object so that it is easy for anyone to understand the code.